### PR TITLE
Issue 15493 content types meta issue v2 - tests

### PR DESCRIPTION
**Pull request by alfredo-dotcms**

### CODE REVIEW

File: src/app/portlets/content-types/fields/content-type-fields-add-row/content-type-fields-add-row.component.scss [hunk 1]
```diff
@@ -30,9 +30,13 @@ $row-height: 70px;
     align-items: baseline;
     height: 198px;
     justify-content: center;
-    text-align: center;
 
-    &.dot-add-rows__add {
+    ::ng-deep .ui-button.ui-button-icon-only {
```
Comment (fmontes) [inline src/app/portlets/content-types/fields/content-type-fields-add-row/content-type-fields-add-row.component.scss]: `_splitbutton.scss`

File: src/app/portlets/content-types/fields/service/field-drag-drop.service.ts [hunk 2]
```diff
@@ -181,16 +181,18 @@ export class FieldDragDropService {
         return this.isDraggingFromSource(source);
     }
 
+    // tslint:disable-next-line:cyclomatic-complexity
```
Comment (fmontes) [inline src/app/portlets/content-types/fields/service/field-drag-drop.service.ts]: The idea of adding that rule into tslint is to use it :)

File: src/app/portlets/content-types/fields/service/field-drag-drop.service.ts [hunk 3]
```diff
@@ -181,16 +181,18 @@ export class FieldDragDropService {
...
     private shouldMoveRow(
         _el: HTMLElement,
         source: HTMLElement,
         handle: HTMLElement,
         _sibling: HTMLElement
     ): boolean {
+        const noDrag = !handle.classList.contains('no-drag');
```
Comment (fmontes) [inline src/app/portlets/content-types/fields/service/field-drag-drop.service.ts]: Did you try to preventDefault or stopProppagation?

File: src/app/portlets/content-types/fields/content-type-fields-add-row/content-type-fields-add-row.component.scss (hunk 1 above)
Comment (alfredo-dotcms) [reply src/app/portlets/content-types/fields/content-type-fields-add-row/content-type-fields-add-row.component.scss]: so in the future no icons will be supported for all splitbuttons in the submenu

File: src/app/portlets/content-types/fields/dot-content-type-fields-variables/components/dot-content-type-fields-variables-table-row/dot-content-type-fields-variables-table-row.component.scss [hunk 4]
```diff
@@ -14,23 +14,40 @@
     .field-variable-row-highlight {
         background-color: $gray-bg;
     }
+
+    p-celleditor {
+        span {
+            display: block;
+            height: 34px;
```
Comment (fmontes) [inline src/app/portlets/content-types/fields/dot-content-type-fields-variables/components/dot-content-type-fields-variables-table-row/dot-content-type-fields-variables-table-row.component.scss]: can this be 32? (8 x 4)?

File: src/app/portlets/content-types/fields/dot-content-type-fields-variables/components/dot-content-type-fields-variables-table-row/dot-content-type-fields-variables-table-row.component.scss [hunk 5]
```diff
@@ -14,23 +14,40 @@
...
+        }
+        input {
+            height: 25px;
```
Comment (fmontes) [inline src/app/portlets/content-types/fields/dot-content-type-fields-variables/components/dot-content-type-fields-variables-table-row/dot-content-type-fields-variables-table-row.component.scss]: can this be 24px?

File: src/app/portlets/content-types/fields/service/field-drag-drop.service.ts (hunk 3 above)
Comment (alfredo-dotcms) [reply src/app/portlets/content-types/fields/service/field-drag-drop.service.ts]: it does not work

**Pull request merged**